2015/presentations/import_deep_dive/text.md

```diff
@@ -0,0 +1,297 @@
+# Import Deep Dive Detours - Petr Viktorin
+
+My talk, Import Deep Dive, gave you a simplified overview of Python's import
+machinery, enough to understand what's going on and where to look for more
+details.
```

Comment:
Think about readers who did not attend your talk, could you provide some introduction here?